lib/DockerWrapper.js

```diff
@@ -284,27 +284,44 @@ module.exports = {
     });
   },
 
-  updateService: (updateConfig) => {
-    return (app) => {
-      return new Promise(async (resolve, reject) => {
-        console.log('Updating service...');
-        const managerNode = await getManagerNodeInstance();
-        const service = managerNode.getService(`${app.title}_web`);
-        const currentServiceConfig = await getCurrentServiceConfig(service);
-
-        const currentServiceSpec = currentServiceConfig.Spec;
-        const versionSpec = { _query: { "version": currentServiceConfig.Version.Index }};
-        const newConfig = Object.assign(currentServiceSpec, updateConfig, versionSpec);
-
-        service.update(newConfig, (error, result) => {
-          if (error) {
-            console.log(error);
-            reject(error);
-          }
-          console.log('Web service updated!');
+  updateService: (app) => {
+    return new Promise(async (resolve, reject) => {
+      const managerNode = await getManagerNodeInstance();
+      const proxyConfig = await Config.findOne({
+        where: { key: 'proxyNetwork' }
+      });
+      
+      const serviceParams = {
```

Comment:
Since these are the same `serviceParams` we use during creation we may just be able to pull the current service params for the service and feed them back into the `.update` call. Not really important to change, but just adding as a note for when we merge this.

Reply:
I'm getting the following when I try to use the params straight from `getCurrentServiceConfig`:
```javascript
statusCode: 400,
  json: {
    message: `invalid service version '': strconv.ParseUint: parsing "": invalid syntax`
```

I can take the name, labels, task template and networks from the received params, but there's needs to be a "_query" key with an object having value { "version": `version` } for the service to update.

I've decided to go with a hybrid, where I ask for service params, and use the ones that are relevant and useable.